main.rb

```diff
@@ -67,6 +67,9 @@ def ask_rental_info(app)
     person_id = ask_for_person_id
     app.list_rentals_by_person(person_id)
   when '7'
+    app.save_books_to_json('local_db/books.json')
+    app.save_people_to_json('local_db/people.json')
+    app.save_rentals_to_json('local_db/rentals.json')
```

Comment:
- [ ] Because the books.json file is empty now the application gives an error when opening the app, please make sure that there is an empty array or remove the file entirely.

Reply:
on it